### PR TITLE
doc/1255_add_documentation_about_running_cygnus

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #1255 
* No tests are required
* Assignee: @frbattid

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -20,3 +20,4 @@
 - [cygnus][hardening] Fix yum install command (#1249)
 - [cygnus][hardening] Add FIWARE architecture section in the documentation (#1248)
 - [cygnus][doc] Fix API entries in documentation (#1246)
+- [cygnus][doc] Add entries about running as process and running as services in documentation (#1255)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: services --> service

Final white line required by Github

File: doc/cygnus-common/installation_and_administration_guide/running_as_service.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#Running Cygnus as a service
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/running_as_service.md]: Cygnus -> cygnus-common

This must be checked in all the entries about running

File: doc/cygnus-common/installation_and_administration_guide/running_as_service.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+**Note**: Cygnus can only be run as a service if you installed it through the RPM.
+
+Once the `cygnus_instance_<id>.conf` and `agent_<id>.conf` files are properly configured, just use the `service` command to start, restart, stop or get the status (as a sudoer):
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/running_as_service.md]: Running cygnus-common is the same than running a plain Flume... this sentence could be used somewhere in this paragrah :)

File: doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#Running Cygnus as a process
```
Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md]: cygnus-ngsi

Comment (pcoello25) [inline doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md]: Fixed in b05b246

File: doc/cygnus-common/installation_and_administration_guide/running_as_service.md (hunk 3 above)
Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/running_as_service.md]: Fixed in b05b246

**Pull request merged**